### PR TITLE
feat(STONEINTG-343): set ResourceQuota for Appstudio CRDs

### PR DESCRIPTION
**Pull request by dirgim**
* Add e2e tests for the new resource quotas

### CODE REVIEW

File: testsupport/tiers/checks.go [hunk 1]
```diff
@@ -511,6 +511,13 @@ func (a *appstudioEnvTierChecks) GetNamespaceObjectChecks(_ string) []namespaceO
 		resourceQuotaComputeDeploy("20", "32Gi", "1750m", "32Gi"),
 		zeroResourceQuotaComputeBuild(),
 		resourceQuotaStorage("50Gi", "50Gi", "50Gi", "12"),
+		resourceQuotaAppstudioCrds("512", "512", "512"),
```
Comment (alexeykazakov) [inline testsupport/tiers/checks.go:514]: Wrong tier. Should be in `func (a *appstudioTierChecks)` Not in `func (a *appstudioEnvTierChecks)`

Comment (dirgim) [reply testsupport/tiers/checks.go:514]: The new resource quotas are supposed to be per-namespace, isn't `appstudioTierChecks` meant for cluster-scoped checks?

Comment (alexeykazakov) [reply testsupport/tiers/checks.go:514]: No. Each tier has its own struct and a set of methods to check different resources in that tier.
You added your checks to the `func (a *appstudioEnvTierChecks) GetNamespaceObjectChecks(_ string)` method. Which returns the checks for namespace scoped objects in the `appstudio-env` tier.
But instead you need to add those checks to the `func (a *appstudioTierChecks) GetNamespaceObjectChecks(_ string)` method which is for the namespace scoped object checks for the `appstudio` tier.

`appstudio` and `appstudio-env`are two different tiers. The former is for appstudio workspaces and the later is for future environments in workspaces (like dev, stage, prod, etc).

Comment (dirgim) [reply testsupport/tiers/checks.go]: Ah, I understand now, thanks. I moved the checks to the correct function now.

**Review by alexeykazakov — APPROVED** [review]
Looks good now. But could you please avoid pushing with force to PR branches in the future? It would help with follow up review when it's enough to review the follow up commits only instead of re-review the entire PR over again.
Thank you! :)

**Pull request merged**